.circleci/config.yml

```diff
@@ -146,3 +145,50 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+  checkForNewEapBuildNumber:
+    docker:
+      - image: 'gitmachete/intellij-plugin-ci:6.1.0'
+    working_directory: ~/git-machete-intellij-plugin
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Reconfigure origin remote to https
+          command: git remote set-url origin https://${GITHUB_TOKEN}@github.com/VirtusLab/git-machete-intellij-plugin.git
+      - run:
+          name: Configure email and name
+          command: |
+            git config user.email "cron_job@example.com"
+            git config user.name "cron_job"
+      - run:
+          # language=sh
+          command: |
+            ./gradlew updateEapBuildNumber
+            if [[ `git status --porcelain` ]]; then
+              newEapBuildNumber=$(git diff intellijVersions.properties | grep -o '+eapOfLatestSupportedMajor.*' | cut -d= -f2)
+              branch=updateEap/$newEapBuildNumber
+              git checkout -b "$branch"
+              git add .
+              git commit -m "New eap build number $newEapBuildNumber"
+              git push origin "$branch"
+              hub pull-request \
+                --no-edit \
+                --base=develop \
+                --labels=ci
+            fi
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
+  updateEapBuildNumber:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
This is _every hour_, right? 🤔

Reply:
I think even once daily is enough

Reply:
no it should run every day at 12:00am UTC